faunadb/strconv.go

```diff
@@ -2,9 +2,19 @@ package faunadb
 
 import (
 	"reflect"
+	"strconv"
 	"strings"
 )
 
+func exprToString(e Expr) string {
+	switch e.(type) {
+	case StringV:
+		return strconv.Quote(e.String())
+	default:
+		return e.String()
+	}
+}
```

Comment:
How is this different than having `StringV` quote itself?

Reply:
The reasoning behind this is detailed in [DRV-195](https://faunadb.atlassian.net/browse/DRV-195)

Reply:
Could you point me to a test that demonstrate this change? I don't understand it from the looks of the code.

Reply:
[#diff-9e9ddfbf28e6188cb1941245667a905bR16](https://github.com/fauna/faunadb-go/pull/107/files#diff-9e9ddfbf28e6188cb1941245667a905bR16)

Reply:
I guess what I'm confused at is: what are the cases where we don't what a `StringV` to be quoted?

Reply:
Could you circle back to this question, please.

Reply:
For some sense of consistency between drivers, I would prefer that strings containing newlines do not escape the newlines, as that requires developers to take an extra step to emit human-readable string output for something like the `Format` example in DRV-195.

There is an alternative. Instead of simply using double quotation marks to delimit string boundaries, it would be totally acceptable to have `StringV` in the output, similar to the C# driver. So: `StringV{abc123}`. Or, with a newline:

```
StringV{
}
```

Reply:
I've decided to keep this as initially suggested by @faunaee  - IMO emitting StringV would clutter the output.

Reply:
I'm sorry to insist but, why can't this logic be solved via polymorphism? This and the `faunaValueToString` are quite counterintuitive in Go.